source/locates.html.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Locates FIX API Specification
+
+
+toc_footers:
+  - [Powered By Slate](https://github.com/slatedocs/slate)
+
+search: true
+---
+
+# Introduction
+Locates FIX API specification details the tags and values along with description of values for each field for processing locate requests and acceptances from OMS clients. Clear Street currently utilizes FIX 4.2 format.
+
+# Configuring FIX Session
+```
+// Example Config
+[DEFAULT]
+ReconnectInterval=5
+SenderCompID=LOCATESCLIENT
+HeartBtInt=30
+SocketAcceptPort=31234
+TargetCompID=CLST
+HeartBtInt=30
+SocketConnectPort=31234
+SocketConnectHost=100.0.0.0
+BeginString=FIX.4.2
+
+[SESSION]
+StartTime=04:00:00
+EndTime=22:00:00
+```
+A FIX session is defined as a unique combination of a `BeginString` (the FIX version number 4.2), a `SenderCompID` (OMS Client ID as defined by Clear Street), and a `TargetCompID`.
+
+A FIX session has an active time period during which all data is transmitted. Clear Street currently accepts connections between 4 AM EST to 10 PM EST on all business days. Clear Street will act as an acceptor and OMS will be the initiator.
+
+A `Logon` message (`Tag 35=A`) must be sent to Clear Street on every business day to indicate the begining of activity and a `Logout` message (`Tag 35=5`)must be sent to Clear Street indicating end of activity for that day. 
+
+Clear Street also recommends exchange of heartbeats every 30 seconds by sending a message with `Tag 35=0`.
+
+Clear Street expects every message sent to have a unique continuous sequence number as part of message with Tag 34=. If there are any gaps in sequence numbers, a sequence reset message will sent to OMS client with tag `34=4`.
+
+# Message List
+
+| Message | Tag 35 | Comments | Direction w.r.t. CLST |
```

Comment:
This isn't properly formatting in the rich text view

Reply:
Thanks. Fixed.